classes/file/FileManager.inc.php

```diff
@@ -543,21 +543,97 @@ function parseFileExtension($fileName) {
 
 	/**
 	 * Decompress passed gziped file.
-	 * @param $filePath string
-	 * @param $errorMsg string
-	 * @return boolean|string
+	 * @param $filePath string The compressed file's filename
+	 * @param $errorMsg string Any fatal error encountered (modified by reference)
+	 * @return boolean|string False on failure, or the decompressed filename on success.
 	 */
 	function decompressFile($filePath, &$errorMsg) {
+		if (extension_loaded('Zlib')) {
+			$chunk = 1024*1024;
```

Comment:
A defined constant would be better at the top of the file, rather than hard-coding a chunk size at a couple points in the code.

Reply:
True.  Will fix.